configuration.py

```diff
@@ -189,17 +189,23 @@ def get_theme(self) -> str:
         the name returned will be the current Windows theme.
         """
         return self.config.get("theme", THEME_DEFAULT)
-
-    def get_voiceattack(self) -> str | None:
+    
+    def get_voiceattack_host(self) -> str:
```

Comment:
Whilst this is configurable the VoiceAttack plugin does not currently support configuring this. This can come in a subsequent plugin release when needed. It will not require a new version of WhisperAttack.